types/wink-jaro-distance/wink-jaro-distance-tests.ts

```diff
@@ -0,0 +1,15 @@
+import jaro = require('wink-jaro-distance');
+
+const result = jaro('abcdef', 'fedcba');
+const result2 = jaro('abcdef', 'fedcba');
+
+// Just need to make sure the types are a number I suppose?
```

Comment:
You can assert types with the `$ExpectType` dtslint feature instead:
```typescript
result.distance; // $ExpectType number
result.similarity; // $ExpectType number
```

Reply:
Oh perfect! I wasn't ware of those features. I'm honestly really really impressed with dtslint. Especially with how it checks the original source code and things